Python/samples/cartpole/sim/render.py

```diff
@@ -128,3 +128,10 @@ def on_draw(self):
         
         draw_cartpole(self.model, self.width, self.height)
 
+    def on_mouse_press(self, x, y, button, modifiers):
+        # print(f"mouse_press {x} {y} {button} {modifiers}")
```

Comment:
Should we remove these for the sample? They seem like debug logs.

Reply:
removed